openhtf/core/test_executor.py

```diff
@@ -60,6 +60,10 @@ def __init__(self, test_descriptor, execution_uid, test_start, test_options):
     self._full_abort = threading.Event()
     self._teardown_phases_lock = threading.Lock()
 
+  def cleanup(self):
```

Comment:
nit, docstring describing when/why to use cleanup vs other things like abort, etc.

Reply:
Done.